aws/ami/windows/scripts/Installers/Install-VS.ps1

```diff
@@ -21,7 +21,7 @@ if (${env:INSTALL_WINDOWS_SDK} -eq "1") {
 }
 
 if (Test-Path "${env:ProgramFiles(x86)}\Microsoft Visual Studio\Installer\vswhere.exe") {
-    $existingPath = & "${env:ProgramFiles(x86)}\Microsoft Visual Studio\Installer\vswhere.exe" -products "Microsoft.VisualStudio.Product.BuildTools" -version "[${env:VS_VERSION}, ${VS_VERSION_major + 1})" -property installationPath
+    $existingPath = & "${env:ProgramFiles(x86)}\Microsoft Visual Studio\Installer\vswhere.exe" -products "Microsoft.VisualStudio.Product.BuildTools" -version "[${env:VS_VERSION}, $($VS_VERSION_major + 1))" -property installationPath
```

Comment:
The former syntax was incorrect. The `"${env:VS_VERSION}, ${VS_VERSION_major + 1}"` expression evaluates to `""`. See [https://www.pdq.com/blog/using-subexpressions-within-strings/](https://www.pdq.com/blog/using-subexpressions-within-strings/) for more details.